readthedocs/config/models.py

```diff
@@ -11,11 +11,24 @@
     'Python',
     [
         'version',
+        'install',
```

Comment:
This is a list, not completely satisfy with this name, any suggestion for rename or if it's fine to let that name are welcome p: